internal/pkg/flags/flag_to_value.go

```diff
@@ -5,13 +5,15 @@ import (
 	"time"
 
 	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
 )
 
 // Returns the flag's value as a string.
 // Returns "" if the flag is not set, if its value can not be converted to string, or if the flag does not exist.
-func FlagToStringValue(cmd *cobra.Command, flag string) string {
+func FlagToStringValue(cmd *cobra.Command, flag string, p *print.Printer) string {
```

Comment:
nitpick: pass the printer as the first argument (similar to the context, which is usually passed everywhere and is always the first argument, when present)

Reply:
this comment applies to the rest of the PR

Reply:
made printer first argument in all files.